src-ui/assets/i18n/ja.json

```
@@ -2144,25 +2170,64 @@
     "title": "アニメーション"
   },
   "status-automations": {
-    "behaviourWarningFAQ": "ステータス変更の挙動に関する質問に対する回答は、こちらをご覧下さい。\n\nQ: VRChat上で青(Join Me)に表示されていなくても、Req Inviteは自動的に承認されますか？\nA: OyasumiVRがステータスを青(Join Me)に変更した場合は、そのタイミングから受け取ったReq Inviteを自動承認します。\n\nQ: ステータスが更新されないのでJoinできません。\nA: ステータスの更新は、他のユーザーに表示されるまでに時間がかかる場合があります。Joinできない場合は、メインメニューからプロフィールを開いて自身のステータスを強制的に更新することも可能です。",
+    "behaviourWarning": "OyasumiVRによるステータス変更は、すぐに反映されない場合があります。これはVRChatのキャッシュによるものであり、時間が少し経てば更新されます。",
+    "behaviourWarningFAQ": "ステータス変更の挙動に関する質問に対する回答は、こちらをご覧下さい。\n\nQ: VRChat上で青(Join Me)に表示されていなくても、Req Inviteは自動的に承認されますか？\nA: OyasumiVRがステータスを青(Join Me)に変更した場合は、受け取ったReq Inviteを自動承認します。\n\nQ: ステータスが更新されないのでJoinできません。\nA: ステータスの更新は、他のユーザーに表示されるまでに時間がかかる場合があります。Joinできない場合は、メインメニューからプロフィールを開いて自身のステータスを強制的に更新することも可能です。",
+    "general": {
+      "description": "特定の状況でVRChatステータスを更新できます。例えば、寝るときや起きるときにステータスを変更します！",
+      "onSleepDisable": {
+        "description": {
+          "status": "ステータスの設定",
+          "statusMessage": "ステータスメッセージの設定"
+        },
+        "script": "睡眠モードを無効化",
+        "title": "起床時"
+      },
+      "onSleepEnable": {
+        "description": {
+          "status": "ステータスの設定",
+          "statusMessage": "ステータスメッセージの設定"
+        },
+        "script": "睡眠モードを有効化",
+        "title": "睡眠時"
+      },
+      "onSleepPreparation": {
+        "description": {
+          "status": "ステータスの設定",
+          "statusMessage": "ステータスメッセージの設定"
+        },
+        "script": "就寝準備",
+        "title": "就寝準備時"
+      }
+    },
     "moreInfo": "詳細",
     "playerLimit": {
       "atLimitOrAbove": {
         "description": {
-          "status": "ステータスの設定"
+          "status": "ステータスの設定",
+          "statusMessage": "ステータスメッセージの設定"
         },
         "title": "上限以上"
       },
       "belowLimit": {
         "description": {
-          "status": "ステータスの設定"
+          "status": "ステータスの設定",
+          "statusMessage": "ステータスメッセージの設定"
         },
         "title": "上限未満"
       },
       "description": "ワールドにいる人数に応じてステータスを自動設定し、Joinできる人数を制限できます。",
+      "disableToEdit": "変更するにはこの機能を一度無効にしてください。",
+      "enable": {
+        "description": "ワールドにいる人数に応じてステータスを変更します",
+        "title": "人数制限の自動化"
+      },
       "onlyIfSleepModeEnabled": "睡眠モード有効時のみ",
       "playerLimit": "人数制限",
-      "playerLimitLabel": "{count} 人のプレイヤー（自分を含む）"
+      "playerLimitLabel": "{count, plural, one {1人のプレイヤー} other {#人のプレイヤー}}（自分を含む）"
```

Since Japanese doesn't support pluralism this is unsupported, so you don't have to do this ^^

You can just change it to this:
```suggestion
      "playerLimitLabel": "{count} 人のプレイヤー（自分を含む）"
```

---

Oh, OK!
Thanks for pointing that out.